stl/inc/yvals_core.h

```diff
@@ -176,6 +176,7 @@
 // P0966R1 string::reserve() Should Not Shrink
 // P1001R2 execution::unseq
 // P1006R1 constexpr For pointer_traits<T*>::pointer_to()
+// P1007R3 std::assume_aligned

```

Comment:
`yvals_core.h` needs to define the feature-test macro mentioned in #47 (in the C++20 section, in sorted order).

Then, it needs to be tested in `tests/std/tests/VSO_0157762_feature_test_macros/test.cpp`.